RepositoryCompiler/CodeModel/CodeParsers/CSharp/CSharpCodeParser.cs

```diff
@@ -209,6 +209,11 @@ private List<CaDETClass> CalculateMetrics(List<CaDETClass> linkedClasses)
                     memberBuilder.CalculateMetrics();
                 }
                 c.Metrics = _metricCalculator.CalculateClassMetrics(c);
+                c.InvokedMethods = new HashSet<CaDETMember>();
+                foreach (var member in c.Members)
+                {
+                    c.InvokedMethods.UnionWith(member.InvokedMethods.ToList());
```

Comment:
Should be moved to the metric calculator. Even if we needed this set down the line, it would be better to have CaDETClass offer a method that dynamically creates this set.